src/transformStateWithClones.js

```diff
@@ -7,7 +7,40 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const updateState = [];
```

Comment:
Maybe its better to use stateIterations or stateVersions or somethin like that insted of updateState.